contracts/common/MiniMeToken.sol

```diff
@@ -1,5 +1,7 @@
 pragma solidity ^0.4.6;
```

Comment:
Should we update this to `^0.4.18` as well?

Reply:
Didn't bother to update anything ERC677 or MiniMe as those are getting replace soon.